CHANGELOG.rst

```diff
@@ -4,6 +4,16 @@ Changelog
 Not Yet Released
 ----------------
 
+- Partial support for non-uniformly sampled data
+- Fixes in the `stingray.simulator` module
```

Comment:
That seems like a very non-descriptive comment: what did you fix?

Reply:
Right, I just created a "lots of bugfixes" point at the end.